app/actors/DocumentationActor.scala

```diff
@@ -78,7 +78,7 @@ object DocumentationActor {
    * @param cacheId The cacheid.
    */
   case class RenderedPage(pageHtml: String, sidebarHtml: Option[String], source: Option[String],
-                          translationContext: TranslationContext, cacheId: String) extends Found[RenderedPage]
+                          translationContext: TranslationContext, cacheId: String, versionInfo: Option[VersionInfo]) extends Found[RenderedPage]
```

Comment:
I think `VersionInfo` should be part of `TranslationContext`.  `TranslationContext` is where all the information about a particular translation of the docs goes, including the available versions, and since the `VersionInfo` is also specific to a particular translation, it should be part of the `TranslationContext` too.  This should also simplify this change, as it means less has to change.